### PR TITLE
ADEN-5880 Check if there is no UAP slot size on FeatureVideo page

### PR DESCRIPTION
**Pull request by v3rth**

### CODE REVIEW

File: src/test/java/com/wikia/webdriver/testcases/adstests/TestAdsAbcdProductsPriority.java [hunk 1]
```diff
@@ -34,10 +36,26 @@ public void adsAppNexusVideoAdIsDisplayedIfAbcdAdIsTargetedOnThisSameArticleOasi
...
+  )
+  public void adsUAPFamilyAdIsNotDisplayedOnFeaturedVideoPage() {
+    AdsBaseObject ads = new AdsBaseObject(driver, FV_PAGE.getUrl());
+    verifyAbcdAdIsNotDisplayed(ads);
+
+    Assert.assertTrue(ads.slotHasSize(AdsContent.TOP_LB, 728, 90));
+    verifyNoUAPSizesInSlot(ads, AdsContent.TOP_LB);
+  }
+
+  private void verifyNoUAPSizesInSlot(AdsBaseObject ads, String slot) {
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/testcases/adstests/TestAdsAbcdProductsPriority.java]: MAJOR Remove this unused private "verifyNoUAPSizesInSlot" method. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AUnusedPrivateMethod)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 3 issues
* ![MAJOR][MAJOR] 3 major

Watch the comments in this conversation to review them.

#### 2 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [AdsBaseObject.java#L354](https://github.com/Wikia/selenium-tests/blob/a13396ef4eb2ef085007d96c7df77592d319708b/src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/adsbase/AdsBaseObject.java#L354): Either log or rethrow this exception. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS1166)
1. ![MAJOR][MAJOR] [AdsBaseObject.java#L434](https://github.com/Wikia/selenium-tests/blob/a13396ef4eb2ef085007d96c7df77592d319708b/src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/adsbase/AdsBaseObject.java#L434): Either log or rethrow this exception. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS1166)

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'

File: src/test/java/com/wikia/webdriver/testcases/adstests/TestAdsAbcdProductsPriority.java [hunk 2]
```diff
@@ -34,10 +36,26 @@ public void adsAppNexusVideoAdIsDisplayedIfAbcdAdIsTargetedOnThisSameArticleOasi
...
   }
 
+  @Test(
+      groups = {"AdsUAPFamilyAdIsNotDisplayedOnFeaturedVideoPage", "AbcdProductPriorityOasis"}
+  )
+  public void adsUAPFamilyAdIsNotDisplayedOnFeaturedVideoPage() {
+    AdsBaseObject ads = new AdsBaseObject(driver, FV_PAGE.getUrl());
+    verifyAbcdAdIsNotDisplayed(ads);
+
+    Assert.assertTrue(ads.slotHasSize(AdsContent.TOP_LB, 728, 90));
```
Comment (bulaszenko) [inline src/test/java/com/wikia/webdriver/testcases/adstests/TestAdsAbcdProductsPriority.java]: I thing This assertion is not needed If we are using verifyNoUAPSizesInSlot

Comment (bulaszenko) [reply src/test/java/com/wikia/webdriver/testcases/adstests/TestAdsAbcdProductsPriority.java]: Discussed off line :)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 3 issues
* ![MAJOR][MAJOR] 3 major

Watch the comments in this conversation to review them.

#### 2 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [AdsBaseObject.java#L354](https://github.com/Wikia/selenium-tests/blob/db44a8b6a7610c8d6813c439417baf6b72fbee45/src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/adsbase/AdsBaseObject.java#L354): Either log or rethrow this exception. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS1166)
1. ![MAJOR][MAJOR] [AdsBaseObject.java#L434](https://github.com/Wikia/selenium-tests/blob/db44a8b6a7610c8d6813c439417baf6b72fbee45/src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/adsbase/AdsBaseObject.java#L434): Either log or rethrow this exception. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS1166)

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'

**Pull request merged**